testing/sdk_tasks.py

```diff
@@ -67,7 +67,7 @@ def fn():
 
 
 def check_tasks_not_updated(service_name, prefix, old_task_ids):
-    sdk_plan.get_deployment_plan(service_name)
+    sdk_plan.wait_for_completed_deployment(service_name)
```

Comment:
@gabrielhartmann You really do not need to wait deployment to complete here. get_deploymeny_plan will wait for plan API, ensuring that scheduler started. 
PS: canary strategy will not automatically complete.